File: Backend/AirlineAPI/Models/FlightInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineAPI.Models
{
    public class FlightInfo
    {
        [Key]
        public string FlightID { get; set; }

        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public string From { get; set; }

        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public string To { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public DateTime Departing { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public DateTime Returning { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public  string Baggage { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public string ClassF { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public int Stops { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public string Duration { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public string CompanyName { get; set; }


        [Required]
        [Column(TypeName = "nvarchar(100)")]
        public int Price { get; set; }

    }
}

File: clover/src/app/entities/flightInfo/flight-info.ts
import { Time } from '@angular/common';
import { Seat } from '../Seat/seat';
import { FlightRate } from '../flightRate/flight-rate';

export class FlightInfo {
flightID:number;
from:string;
to:string;
departing:string;
returning:string;
classf:string;
baggage : string;
stops:number;
duration:string;
companyName:string;
price:number;
seatsNumber:number;
rateFlight:number;
companyAboutAvioCompID: number;
userDetailUserId: number;
seatsList: Seat[];
startTime:string;
endTime:string;
flightRates:FlightRate[];

constructor(from:string, to:string,departing:string,returning:string,classf:string,stops:number,duration:string,companyName:string,price:number,seatsNumber:number,
    rateFlight:number){
    this.from=from;
    this.to=to;
    this.departing=departing;
    this.returning=returning;
    this.classf=classf;

    this.stops=stops;
    this.duration=duration;
    this.companyName=companyName;
    this.price=price;
    this.seatsNumber=seatsNumber;
    
}
}
